pkg/cluster/external.go

```diff
@@ -3,21 +3,20 @@ package cluster
 import (
```

Comment:
If I understand it correctly, this file has the logic to manage and unmanage external clusters using the new profiles, is it?

Reply:
No, it uses old profiles.